From b57c67c25f58596638c4f721f2c75a187873c9ea Mon Sep 17 00:00:00 2001
From: Jun Li <r65092@freescale.com>
Date: Wed, 21 Jan 2009 15:16:35 +0800
Subject: [PATCH] ENGR00105179 USBPHY_VBUS source enable and PWR/OC pins config for MX51.

1.Enable SWBST for USBPHY_VBUS.
2.Configure Power and OC pins for USBOTG.

Signed-off-by: Li Jun <r65092@freescale.com>
---
 arch/arm/plat-mxc/utmixc.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-mxc/utmixc.c b/arch/arm/plat-mxc/utmixc.c
index 2e9c639..f4f9f07 100644
--- a/arch/arm/plat-mxc/utmixc.c
+++ b/arch/arm/plat-mxc/utmixc.c
@@ -20,7 +20,6 @@
 #include <linux/err.h>
 #include <linux/platform_device.h>
 #include <linux/fsl_devices.h>
-#include <linux/pmic_external.h>
 #include <linux/usb/fsl_xcvr.h>
 #include <linux/pmic_external.h>
 
-- 
1.5.4.4

